src/transformers/__init__.py

```diff
@@ -1597,6 +1597,7 @@
             "FlaxRobertaPreTrainedModel",
         ]
     )
+    _import_structure["models.t5"].extend(["FlaxT5ForConditionalGeneration", "FlaxT5Model"])
```

Comment:
Missing the PreTrainedModel, but I'm adding it in #12314

Reply:
ok not adding it here then